src/Controller/ShippingGatewayController.php

```diff
@@ -22,11 +24,11 @@
 {
     /**
      * @param Request $request
-     * @param string $template
+     * @param string|null $template
      *
      * @return Response
      */
-    public function getShippingGatewaysAction(Request $request, $template)
+    public function getShippingGatewaysAction(Request $request, ?string $template): ?Response
```

Comment:
I guess the $template is also required and the Response once again can not be nullable.